src/main/java/no/nav/veilarbarena/repository/entity/OppfolgingsbrukerEntity.java

```diff
@@ -24,5 +24,6 @@ public class OppfolgingsbrukerEntity {
     Boolean erDoed;
     ZonedDateTime doedFraDato;
     ZonedDateTime timestamp; // Når brukeren sist ble endret
+    String personId;
```

Comment:
Siden ny query kun henter ut `personId` så tenker jeg String kan returneres direkte i stede for å utvide `OppfolgingsbrukerEntity`. Evt legge det til i row mapperen til `OppfolgingsbrukerEntity` i tillegg, men siden `personId` ikke brukes ellers er det kanskje ikke nødvendig.

Reply:
Det er et godt poeng. Enig i at vi bare kan hente den ut direkte. Jeg fikser det.